behave_test_automation/pah_behave/features/login-testcase.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Login view
```

Comment:
File name should be lowercased with `_` not `-`, just to stick with a one convention.

Reply:
Changed the filename to `login_view`